aioqiwi/kassa/types/invoice.py

```diff
@@ -39,7 +39,7 @@ class Invoice(BaseModel):
     pay_url: str = Field(..., alias="payUrl")
     amount: Amount = Field(..., alias="amount")
     status: Status = Field(..., alias="status")
-    customer: Customer = Field(..., alias="customer")
+    customer: Optional[Customer] = Field(type(None), alias="customer")
```

Comment:
```suggestion
    customer: Optional[Customer] = Field(None, alias="customer")
```

`type(None)` is an old type generator's issue